src/App.tsx

```diff
@@ -1,41 +1,93 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getUsers } from './api/users';
```

Comment:
I see no such file in this pull request. Seems like it was not included, so I suggest you add it manually. The idea is nice 🔥